cmd/detectExecuteScan_test.go

```diff
@@ -66,6 +66,26 @@ func TestRunDetect(t *testing.T) {
 		assert.Equal(t, expectedScript, utilsMock.Calls[0])
 	})
 
+	t.Run("success case - with report", func(t *testing.T) {
+		t.Parallel()
+		utilsMock := newDetectTestUtilsBundle()
+		utilsMock.AddFile("detect.sh", []byte(""))
+		utilsMock.AddFile("my_BlackDuck_RiskReport.pdf", []byte(""))
+		err := runDetect(detectExecuteScanOptions{}, utilsMock)
```

Comment:
```suggestion
		err := runDetect(detectExecuteScanOptions{FailOn: []string{"BLOCKER"}}, utilsMock)
```
Since the generation of json file will now depend on the policy check parameter, this should be provided in the test case